the-event-loop/interval.js

```diff
@@ -0,0 +1,14 @@
+let count = 3;
+
+let decrement = () => {
```

Comment:
You don't change the `decrement` variable so make it a `const`. Remember, your fingers should type `const` first! Then change to `let` only if you have to.